spec/features/admin/shelters/index_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.describe "Admin Shelter Page", type: :feature do
+
+  describe "As a visitor" do
+    describe "When I visit '/admin/shelters'" do
+      before(:each) do
+        @shelter_1 = Shelter.create!(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
+        @shelter_2 = Shelter.create!(name: "Lakewood shelter", city: "Lakewood, CO", foster_program: true, rank: 7)
+        @shelter_3 = Shelter.create!(name: "Westminster shelter", city: "Westminster, CO", foster_program: false, rank: 2)
+
+        @application_1 = Application.create(name: "Julie Johnson", street_address: "201 Main Street", city: "Seattle", state: "WA", zip_code: "75250", description: "I love dogs!", status: "In Progress")
+
+        @pet_1 = Pet.create!(adoptable: true, age: 1, breed: "doberman", name: "Auggie", shelter_id: @shelter_1.id)
+      end 
+
+      #User Story 10
+      it "Then I see all Shelters in the system listed in reverse alphabetical order by name" do
+        visit "/admin/shelters"
+
+        expect(page).to have_content(@shelter_1.name)
+        expect(page).to have_content(@shelter_2.name)
+        expect(page).to have_content(@shelter_3.name)
+
+        expect(@shelter_3.name).to appear_before(@shelter_2.name)
+        expect(@shelter_2.name).to appear_before(@shelter_1.name)
+      end
```

Comment:
Nice job using the `orderly gem` to test how things are showing up on your page.